orc/src/main/java/org/apache/iceberg/orc/OrcSchemaVisitor.java

```diff
@@ -83,9 +112,53 @@
     return visitor.record(record, names, visitFields(fields, names, visitor));
   }
 
-  public void beforeField(String name, TypeDescription type) {}
+  public String elementName() {
+    return "_elem";
```

Comment:
Why underscores?

Reply:
These are the ORC names for list elements. Same for Map as `_key` and `_value`.